src/commonmark-rules.js

```diff
@@ -2,6 +2,17 @@ import { repeat } from './utilities'
 
 var rules = {}
 
+rules.text = {
+  filter: '#text',
+
+  replacement: function (content, node, options) {
+    if (node.isCode) return node.nodeValue
+    return options.escapes.reduce(function (accumulator, escape) {
+      return accumulator.replace(escape[0], escape[1])
+    }, node.nodeValue).trim()
+  }
+}
```

Comment:
Slight concern here is that it requires a fair bit of knowledge to add to the behaviour i.e. a developer has to remember to:
- pass through unchanged code content
- iterate over the existing escapes and performs the replacements
- trim the value